Library/Homebrew/cleanup.rb

```diff
@@ -76,6 +78,13 @@ def stale_gh_actions_artifact?(pathname, scrub)
         scrub || prune?(pathname, GH_ACTIONS_ARTIFACT_CLEANUP_DAYS)
       end
 
+      ATTESTATION_CLEANUP_DAYS = 3
+
+      sig { params(pathname: Pathname, scrub: T::Boolean).returns(T::Boolean) }
+      def stale_attestation?(pathname, scrub)
+        scrub || prune?(pathname, ATTESTATION_CLEANUP_DAYS)
```

Comment:
Might be nice if this could be cleaned up based on the state of the cached bottles? Not a blocker, very much nice-to-have.